src/libsystemd-network/lldp-tlv.h

```diff
@@ -76,6 +87,7 @@ int tlv_packet_append_u32(tlv_packet *m, uint32_t data);
 int tlv_packet_append_string(tlv_packet *m, char *data, uint16_t size);
 
 int lldp_tlv_packet_enter_container(tlv_packet *m, uint16_t type);
+int lldp_tlv_packet_enter_container_org(tlv_packet *m, const uint8_t *oui, uint8_t subtype);
```

Comment:
> "org"? Please no abbreviation, except for local variable declarations.

Well, lldp_tlv_packet_enter_organizationally_specific_container() seemed too long. How about one of the following?

 lldp_tlv_packet_enter_container_oui()
 lldp_tlv_packet_enter_container_oui_subtype()